src/main/java/net/teumteum/user/controller/UserController.java

```diff
@@ -53,6 +40,12 @@ public UsersGetByIdResponse getUsersById(@RequestParam("id") String userIds) {
         return userService.getUsersById(parsedUserIds);
     }
 
+    @GetMapping("/me")
```

Comment:
```suggestion
    @GetMapping("/users")
```

GET /users 에 겹치는게 없으면 이렇게 바꾸는건 어떤가요?